frontends/web/src/routes/account/summary/accountssummary.tsx

```diff
@@ -87,7 +89,7 @@ class AccountsSummary extends Component<Props, State> {
                 ? accountPerCoin[account.coinCode].push(account)
                 : accountPerCoin[account.coinCode] = [account];
             return accountPerCoin;
-        }, {});
+        }, {} as TAccountCoinMap);
```

Comment:
this invocation of `reduce` looks a bit sketchy. the accumulator is modified also returned... seems like a for loop would be cleaner and simpler

Reply:
💯  agree, but should I refactor in this PR, add a comment or do nothing?